src/components/DocumentationTopic/TopicsLinkCardGrid.vue

```diff
@@ -10,7 +10,7 @@
 
 <template>
   <div class="TopicsLinkCardGrid">
-    <Row :columns="compactCards ? 3 : 2">
+    <Row :columns="gridSize">
```

Comment:
What you have here seems pretty clean but do you think this could be done without the use of `BreakpointEmitter`?

I'm thinking like utilizing one of our breakpoint Sass mixins for the right media query to override the `--col-count` variable for the row?

Maybe it's not possible or is just too messy, but I was just wondering out loud since adding the JS emitter might be a little heavyweight for things that can be solved just with CSS.

Reply:
It's ok if the answer is no—just curious.

Reply:
We can deff do that, was just not sure if you would prefer to use the "official" component API, aka the prop, or to try and modify the css vars :) 

We use the breakpoint emitter in a few places, so I thought it might make sense to just have the current breakpoint globally available.

Reply:
My initial thought was that the emitter component just adds additional JS complexity, and I think we're only currently using it in places where CSS alone won't suffice (but I could be wrong).

It could end up being the case that the CSS approach is actually worse when compared to this. I was mostly just wondering if you had considered it and your thoughts.

If we do add it globally, we may want to consider refactoring other places using it so that we don't have multiple emitters going on the same page that do the same thing.